content/videos/setuprailsapm.md

```diff
@@ -0,0 +1,18 @@
+---
+title: APM - Setup for a Rails app
+kind: video
+language: en
+wistiaid: ps2vn2rask
+tags:
+    - Datadog 101
+    - APM
+    - Rails
+summary: Datadog APM is incredibly easy to setup. This video shows you how you can be up and running in just a few minutes with any Ruby on Rails application.
+
+
+---
+Datadog APM is incredibly easy to setup. This video shows you how you can be up and running in just a few minutes with any Ruby on Rails application.
```

Comment:
nit: two quick 'you's in succession. suggestion: "This video shows how you can get up and running.."